specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-06-01/consumption.json

```diff
@@ -1014,6 +1014,55 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.consumption/reservationTransactions": {
```

Comment:
The provider `Microsoft.consumption`  is lower case of `c` while on the response it is `C`

Not sure if this affect the back end service. Anyway, the naming should be consistent.